pkg/sdk/common_types_test.go

```diff
@@ -107,3 +107,177 @@ func TestToBoolProperty(t *testing.T) {
 		assert.Equal(t, prop.Description, row.Description)
 	})
 }
+
+func TestToStorageSerializationPolicy(t *testing.T) {
+	testCases := []struct {
+		Name     string
+		Input    string
+		Expected StorageSerializationPolicy
+		Error    string
+	}{
+		{
+			Name:     "optimized storage serialization policy",
+			Input:    string(StorageSerializationPolicyOptimized),
+			Expected: StorageSerializationPolicyOptimized,
+		},
+		{
+			Name:     "compatible storage serialization policy",
+			Input:    string(StorageSerializationPolicyCompatible),
+			Expected: StorageSerializationPolicyCompatible,
+		},
+		{
+			Name:  "validation: incorrect storage serialization policy",
+			Input: "incorrect",
+			Error: "unknown storage serialization policy: incorrect",
+		},
+		{
+			Name:  "validation: empty input",
+			Input: "",
+			Error: "unknown storage serialization policy: ",
+		},
+		{
+			Name:     "validation: lower case input",
+			Input:    "optimized",
+			Expected: StorageSerializationPolicyOptimized,
+		},
+	}
+
+	for _, testCase := range testCases {
+		t.Run(testCase.Name, func(t *testing.T) {
+			value, err := ToStorageSerializationPolicy(testCase.Input)
+			if testCase.Error != "" {
+				assert.Empty(t, value)
+				assert.ErrorContains(t, err, testCase.Error)
+			} else {
+				assert.NoError(t, err)
+				assert.Equal(t, testCase.Expected, value)
+			}
+		})
+	}
+}
+
+func TestToLogLevel(t *testing.T) {
+	testCases := []struct {
+		Name     string
+		Input    string
+		Expected LogLevel
+		Error    string
+	}{
+		{
+			Name:     "trace log level",
```

Comment:
This is example of a test where IMO being so explicit hurts the readability. The name is totally redundant here, and these could be one-lined. Check e.g. `Test_Warehouse_ToWarehouseSize`. It's much smaller and more readable.

Reply:
Tests shortened